src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -132,11 +132,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $domain = $input->getOption('domain');
         /** @var KernelInterface $kernel */
         $kernel = $this->getApplication()->getKernel();
-        $rootDir = $kernel->getContainer()->getParameter('kernel.root_dir');
```

Comment:
``$rootDir = $kernel->getContainer()->hasParameter('kernel.root_dir') ? $kernel->getContainer()->getParameter('kernel.root_dir') : null`
then compare to `null` below?